### ISSUE TITLE
Suppose moving towards system configuration with flakes.

### ISSUE THREAD

**Issue by gwimm** [bdsm/bdsm#1]
Hey bdsm,
use flakes.

**Comment by bdsm** [comment]
> Hey bdsm,
> 
> use flakes.

you're not wrong. perhaps on my next nixos system; I mainly keep that here for my own reference as I don't have it running as a primary OS on any of my laptops these days. It served a somewhat ephemeral role back when I routinely swapped that drive into machines with different hardware and configuration requirements (being able to simply select a different boot entry with the necessary configuration upon doing so was very useful). I'll leave this open as a reminder to myself.
